atomic_reactor/plugin.py

```diff
@@ -467,9 +467,9 @@ def run(self, *args, **kwargs):
         else:
             logger.debug("No Dockerfile path has been specified")
 
-        kwargs['buildstep_phase'] = True
```

Comment:
> shouldn't this be buildstep_phase=buildstep_phase ?

@mkosiarc Here.

Reply:
I see, what is the reason we are not having this as default in the function?